.github/autobump.txt

```diff
@@ -984,6 +984,7 @@ grpc
 grpc-swift
 grype
 gsettings-desktop-schemas
+gsl
```

Comment:
This plus `no long build conflict` is preventing other PRs from being merged.

See:
https://github.com/Homebrew/homebrew-core/pull/173471
https://github.com/Homebrew/homebrew-core/pull/173469
https://github.com/Homebrew/homebrew-core/pull/173468

Reply:
indeed, should be good now.